css/src/components/image.scss

```diff
@@ -0,0 +1,44 @@
+$dimensions: 16 24 32 36 48 64 76 88 96 112 128 !default;
+
+.image {
+	display: block;
+	position: relative;
+
+	img,
+	svg {
+		display: block;
+		width: 100%;
+		height: auto;
+		max-height: 100%;
```

Comment:
Is it needed? 🤔

Reply:
Likely a browser compat thing, I think we should keep it

Reply:
ok, just wanted to double check :)